compiler_gym/envs/cgra/architectures/CGRA.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Dict, List
+from compiler_gym.envs.cgra.compile_settings import CGRACompileSettings
+
+"""
+This is the most abstract representation of a CGRA ----
+a list of nodes, with an interconnect archtiecture.
+
+This can be inherited from to make things easier.
+"""
+class CGRA(object):
```

Comment:
Docstrings need to be placed directly after the class/function/method declaration:

```py
class CGRA:
    """blah blah"""
    ...
```

Don't put the docstring before, and don't use `#` comments instead of `"""` docstrings. This applies several places else in the PR.

Reply:
Nitpick: don't subclass `object`, it's not necessary (again, applies elsewhere in the PR)